src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -17,8 +17,9 @@ import { getChannelLogger, getDebugPort, localize } from '../utilities/vsCodeUti
 
 import { getLogger } from '../logger'
 import { DefaultValidatingSamCliProcessInvoker } from '../sam/cli/defaultValidatingSamCliProcessInvoker'
+import { normalizeSeparator } from '../utilities/pathUtils'
 import { CodeLensProviderParams, getInvokeCmdKey, getMetricDatum, makeCodeLenses } from './codeLensUtils'
-import { LambdaLocalInvokeParams, LocalLambdaRunner } from './localLambdaRunner'
+import { getHandlerRelativePath, LambdaLocalInvokeParams, LocalLambdaRunner } from './localLambdaRunner'
```

Comment:
So you've removed the class-based code to get the handler path while you're using this value from the functional implementation...I wonder, does the premise of this PR affect the Python and C# implementations or just the JS one? If it's just the JS one, would it be better instead to use this as our excuse to finally move JS over to the functional implementation? Otherwise, this blurs the lines between the two implementations.

Reply:
Essentially, the class based code in `generateInputTemplate` was moved to the CodeLens generator. `generateInputTemplate` has no non-class based equivalent (because the other callers do this prior to calling `makeInputTemplate`)